demo/public/app.js

```diff
@@ -47,7 +47,9 @@ var uiConfig = {
     firebase.auth.EmailAuthProvider.PROVIDER_ID
   ],
   // Terms of service url.
-  'tosUrl': 'https://www.google.com'
+  'tosUrl': 'https://www.google.com',
+  // Whether the display name should be displayed in Sign Up page.
+  'shouldDisplayName': false
```

Comment:
As we discussed offline, it may be better to move that configuration under the `signInOptions` parameter, as it is specific to the `EmailAuthProvider`.  This means adding a new provider specific option, alongside `scope`. See [here](https://github.com/firebase/firebaseui-web/blob/master/javascript/widgets/config.js#L217) how we handle the config per provider.